source/includes_main/_automations.md

```diff
@@ -0,0 +1,147 @@
+# Automations
+## The automation object
+```
+# EXAMPLE OBJECT
+```
+
+```json
+{
+  "data": {
+    "id": "1",
+    "type": "automations",
+    "attributes": {
+      "name": "Public API Automation Example",
+      "enter_triggers": [
+        "prospect.create"
+      ],
+      "exit_triggers": [
+        "prospect.replied"
+      ],
+      "status": "on",
+      "send_as_thread": true,
+      "sending_days": [
+        "monday",
+        "tuesday",
+        "wednesday",
+        "thursday",
+        "friday"
+      ],
+      "start_sending_minutes": 480,
+      "end_sending_minutes": 1020,
+      "timezone": "Etc/UTC",
+      "created_at": "2020-01-31T12:00:00.000Z",
+      "updated_at": "2020-01-31T12:00:00.000Z"
+    },
+    "relationships": {
+      "steps": {
+        "data": [
+          {
+            "id": "abc123",
+            "type": "automations-steps"
+          },
+          {
+            "id": "abc456",
+            "type": "automations-steps"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### Object attributes
+Attribute | Filterable? | Description
+--------- | ----------- | -----------
+id | no | **integer** <br />A unique identifier for the automation
+name | no | **string** <br />The automation's name
+enter_triggers | no | **array** <br />An array containing the conditions required by the prospect to enter the automation. Possible values are `message.sent`, `message.open`, `message.click`, `prospect.create`, `prospect.updated`, `prospect.replied`, `prospect.qualified` and `prospect.convert`
+exit_triggers | no | **array** <br />An array containing the conditions required by the prospect to exit the automation. Possible values are `message.click`, `prospect.replied`, `prospect.qualified`, `prospect.convert`, `message.hard_bounced` and `pseudo_event.unmatch`
+status | **yes** | **string** <br />The automation's status that can take 3 different values: `on`, `off` and `archived`
+send_as_thread | no | **boolean** <br />The automation's emails should be sent as threads.
+sending_days | no | **array** <br />An array containing the days of the week when the automation will send.
```

Comment:
An array containing days during which emails can be sent ("monday", "tuesday", etc.)